src/plasmapy/formulary/collisions/misc.py

```diff
@@ -325,6 +334,157 @@ def Bethe_stopping(
     return Bethe_stopping_lite(I.si.value, n.si.value, v.si.value, z) * u.J / u.m
 
 
+def _f_n_mol_integrand(
+    u: float,
+    ϑ: float,
+    n: int,
+):
+    """Eq. 26 of Bethe."""
+    # TODO: Move this into the body of `Bethe_Ferrari_Moliere_scattering`
+    return u * jv(0, ϑ * u) * np.exp(-(u**2) / 4) * (u**2 / 4 * np.log(u**2 / 4)) ** n
+
+
+def _f_mol_n(
+    ϑ: u.Quantity[u.dimensionless_unscaled],
+    n: int,
+):
+    integral = [
+        quad(_f_n_mol_integrand, 0, np.inf, args=(x, n))[0] for x in ϑ
+    ] * u.dimensionless_unscaled
+
+    return integral / factorial(n)
+
+
+def Bethe_Moliere_scattering(
```

Comment:
I always just hear this called Moliere scattering- I think Bethe mostly just described Moliere's results in english?